site/docs/guide/deployment-types.md

```diff
@@ -1,5 +1,6 @@
 ---
 prev: ./games.md
+next: ./telegram-payments.md
```

Comment:
I'd rather have the payments page after games but before deployment types. Deploying is what happens last.